olusola-afikode.js

```diff
@@ -0,0 +1,8 @@
+const firstName = "Olusola";
+const lastName = "Afikode";
+const fullName = firstName + " " + lastName;
+const id = "HNG-06501";
+const email = "afikodeolaitan@gmail.com";
+const languageUsed = "JavaScript";
+
+const myDetails = `Hello World, this is ${fullName} with HNGi7 ID ${id} and email ${email} using ${languageUsed} for stage 2 task`;
```

Comment:
you need to console.log your details.kindly do so.

Reply:
It was an oversight on my part. I was also struggling with how to use some git commands